### ISSUE TITLE
10$: warn on semicolon after JSX component

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1105]
```javascript
return <div>
  <b>hello</b>;
</div>;
```

quick-lint-js should warn for the first semicolon.

Criteria:
* Inside a JSX element
* Immediately after a JSX element
* Single semicolon followed by newline
* No space between `>` (end of JSX element) and semicolon (maybe?)
